sdk/tests/conformance/extensions/00_test_list.txt

```diff
@@ -26,6 +26,7 @@
 --min-version 1.0.3 --max-version 1.9.9 oes-texture-half-float-with-image.html
 --min-version 1.0.3 --max-version 1.9.9 oes-texture-half-float-with-video.html
 --min-version 1.0.2 --max-version 1.9.9 oes-element-index-uint.html
+--min-version 1.0.4 --max-version 1.9.9 oes-fbo-mipmap-renderer.html
```

Comment:
Typo here: should be the name of the HTML file, e.g. oes-fbo-render-mipmap.html . Could you please put up a follow-on pull request? Thanks.

Reply:
Thanks for the catch Ken. This actually lead to test bot failure. Pull request is here: 2954